tests/gocase/util/server.go

```diff
@@ -59,8 +60,13 @@ func (s *KvrocksServer) NewTCPClient() *tcpClient {
 }
 
 func (s *KvrocksServer) Close() {
-	require.NoError(s.t, s.cmd.Process.Kill())
-	require.EqualError(s.t, s.cmd.Wait(), "signal: killed")
+	require.NoError(s.t, s.cmd.Process.Signal(syscall.SIGTERM))
+	// A Pod is granted a term to terminate gracefully, which defaults to 30 seconds.
```

Comment:
What does `Pod` stand for here? It seems a comment copied from somewhere that doesn't fit this situation.

Reply:
Kubernetes will send a SIGTERM signal to the containers in the pod after deleting the pod. It waits for a specified time, called the termination grace period. By default, this is 30 seconds. If the containers are still running after the grace period, they are sent the SIGKILL signal and forcibly removed.

Reply:
So here is a local integration test environment. For nothing, we should talk about Kubernetes things?

Reply:
Yeah, it is two separate things. I just want the wait time not to be a magic number.

Reply:
Users may want to exit gracefully in 30 seconds, so I change it like this.